hnginternship5/ziki
# Published posts pages redesigned for desktop, tablet and mobile

###PURPOSE OF PR
To create the published posts page

###FEATURES IMPLEMENTED
Redesigned the page to fit the design from Figma and added responsiveness.

###HOW TO TEST
Open the page on any device to see the responsive fit.

###PIVOTAL TRACKER STORY LINK
https://www.pivotaltracker.com/story/show/165477958


![Screenshot_2019-04-20 Screenshot](https://user-images.githubusercontent.com/33119163/56448639-a0318980-6308-11e9-9a40-41c075f386ab.png)
![Screenshot_2019-04-20 Screenshot(2)](https://user-images.githubusercontent.com/33119163/56448640-a0ca2000-6308-11e9-80df-5fedb253aa52.png)
![Screenshot_2019-04-20 Screenshot(1)](https://user-images.githubusercontent.com/33119163/56448641-a162b680-6308-11e9-8ddb-ffe28f5cebde.png)
